public/pages/DetectorDetail/containers/DetectorDetail.tsx

```diff
@@ -226,6 +227,7 @@ export const DetectorDetail = (props: DetectorDetailProps) => {
   const handleStartAdJob = async (detectorId: string) => {
     try {
       await dispatch(startDetector(detectorId));
+      dispatch(getDetector(detectorId));
```

Comment:
`await` is used to wait for the promise returned after calling `startDetector`, to make sure the detector was started correctly, so we can show the success toast. We don't care about waiting for the `getDetector` call here, and instead will just make the call, and update the page once the call returns at some later time.

The reason for adding this line here at all, is because after starting the detector, we need to re-fetch the latest RT and historical tasks to show the latest info.

Reply:
Could you add these to the comments?

Reply:
Sure, will add.